test_net2.yaml.template

```diff
@@ -9,3 +9,4 @@ spec:
   prefix: "24"
   status: "Init"
   bouncers: 1
+  external: True
```

Comment:
what's the default behavior if this is not specified?

Reply:
The subnet's external is false